lib/restbase.js

```diff
@@ -33,12 +33,13 @@ function cloneRequest(req) {
     };
 }
 
-function RESTBase(options, req) {
+function RESTBase(options, req, parOptions) {
```

Comment:
I considered that option too, but then decided against it. I realized that we still need to support implicit child creation so that the restbase instance for sub-requests has the proper context. We'd have to pass a flag to indicate that we don't want to create another child in the special created-child-in-server-case, or figure out some other criteria to base the decision on.